Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -53,18 +53,18 @@ source file contains mixed test content.
 
 ### Convert test classes
 
-XCTest groups related sets of test methods in test classes: classes that inherit
+XCTest groups related sets of test methods in test classes that inherit
```

Comment:
This change is incorrect. We're saying "XCTest groups related sets of test methods in test classes", then we're defining what "test classes" are.

Reply:
Will revert.